container-runtime/stargz-snapshotter/stargz-snapshotter.yaml

```diff
@@ -0,0 +1,21 @@
+name: stargz-snapshotter
+container:
+  entrypoint: ./containerd-stargz-grpc
```

Comment:
as this service is using `/var`, it should made dependent on `cri` service to guarantee it starts when `/var` is mounted and stopped before it gets unmounted

Reply:
Looks like it still works with this dep.

Reply:
I'm confused how the system containerd behaves in this case, since both cri and containerd uses the same config